src/dashboard/Settings/reducer.js

```diff
@@ -0,0 +1,81 @@
+import {
+  STORE_INCIDENTS_TYPE,
+  SELECTED_INCIDENT_TYPE,
+  TRIGGER_GET_INCIDENTS_TYPE,
+  ADD_NEW_INCIDENT_TYPE,
+  UPDATE_INCIDENT_TYPE,
+  COLOR_AUTOFILL,
+} from './actions';
+
+/**
+ * Incident reducers
+ *
+ * @function
+ * @name IncidentsType
+ *
+ * @param {Object} state - Initial state
+ * @param {Object} action - Redux action object
+ * @param {string} action.type - Action type
+ *
+ * @version 0.1.0
+ * @since 0.1.0
+ * */
+
+const initialState = {
+  filters: [],
+  data: [],
+  total: 0,
+  isLoading: false,
+  error: null,
+};
+
+export default function incidentsType(state = initialState, action) {
```

Comment:
I suggest breaking down reducers based on what part of the state they are affecting/mutating